BenchmarkVCFs/BenchmarkVCFs.wdl

```diff
@@ -96,16 +98,17 @@ workflow Benchmark {
                 message="Variant selector list is length "+length(actualSelectorJEXL)+" while labels list is "+length(actualSelectorLabels)
         }
     }
-
-    call MatchEvalTruth as Match {
-        input:
-            evalVcf=evalVcf,
-            truthVcf=truthVcf,
-            evalVcfIndex=evalVcfIndex,
-            truthVcfIndex=truthVcfIndex,
-            hapMap=hapMap,
-            gatkTag=gatkTag,
-            preemptible=preemptible
+    if (fingerprint_toggle){
```

Comment:
I'd put a space between the `)` and the `{` so `(run_fingerprint) {` instead

Reply:
*{,  not }, for the commit message below!